src/i18n/es.js

```diff
@@ -27,6 +27,19 @@ const urlSiteContact = `${urlSite}/contacto/`;
 const urlControlPanelMain = `${urlControlPanel}/ControlPanel`;
 
 export default {
+  campaign_history: {
```

Comment:
```suggestion
  subscriber: {
```

Reply:
What do we do with this? in the next PR when you add the icon into the grid will be ok too.